README.md

```diff
@@ -52,6 +52,12 @@ As a convenience, the following commands override some config for you:
 
 Visit `localhost:3001` to see the UI locally.
 
+### API Responses
+
+There is a sample of the API responses that we receive from Platform in `sampleApiResponseStructure.json`. It is abbreviated but shows how we receive filters and
+searchresults.
```

Comment:
should have a space between search and results

Reply:
👍